lib/template/details-html.php

```diff
@@ -161,6 +161,8 @@ function _one_keyword_row($keyword_token,$word_id){
 					if ($aPointDetails['wikipedia'])
 					{
 						kv('Wikipedia Calculated' , wikipedia_link($aPointDetails) );
+					} else if ($aPointDetails['aExtraTags']['wikipedia']) {
+						kv('Wikipedia Calculated' , wikipedia_link($aPointDetails['aExtraTags']) );
```

Comment:
Ok! Then originally tagged values should have a different title, like 'Wikipedia'?